crates/xilem_web/src/svg/kurbo_shape.rs

```diff
@@ -29,32 +29,36 @@ impl ViewMarker for Line {}
 impl Sealed for Line {}
 
 impl<T, A> View<T, A> for Line {
-    type State = VecMap<Cow<'static, str>, AttributeValue>;
+    type State = (
+        VecMap<Cow<'static, str>, AttributeValue>,
+        VecMap<Cow<'static, str>, ()>,
+        VecMap<Cow<'static, str>, Cow<'static, str>>,
+    );
```

Comment:
Maybe we can create extra types (`ElementState`) for (sub) states of `View`s implementing e.g. the `Element` trait, to avoid doing all this for all kinds of types implementing these interfaces?

Reply:
Yep let's do that.